app/tutorial_servo_control/src/servo_control.cpp

```diff
@@ -0,0 +1,32 @@
+#include <servo_control.h>
+#include "mbed.h"
+
+TutorialServo::setAngleRangeInDegrees(float degrees)
+{
+    m_servoRangeInDegrees = degrees;
+}
+
+TutorialServo::setPositionInDegrees(float degrees)
+{
+    if (degrees > m_servoRangeInDegrees)
+    {
+        //exit beforing attempting to set position greater than set max
+        return;
+    }
+    //convert degrees into pulse width
+    //the position is determined by the percentage that the value is out of 180 degrees eg. 90 is 1.5ms while 180 is 2ms
+    float pulseWidth = 0.001 + 0.001*degrees/180;

```

Comment:
Your minimum and maximum pulsewidths are what you should be using to scale your input. This is since all servos will have a little bit of a different range, and you want to use this instead of the standard 1-2ms range.